src/meta/src/stream/stream_manager.rs

```diff
@@ -685,6 +685,357 @@ where
 
         Ok(())
     }
+
+    pub async fn create_sink(
```

Comment:
Please avoid duplicating so many codes.

Reply:
Apart from `fn get_dependent_relations` I also introduced the `fn get_internal_table(&self, ctx: &CreateMaterializedViewContext) -> RwResult<Vec<Table>>` method to reduce code duplication in `fn create_sink`, `create_materialized_view` and `create_materialized_source_inner`.